docs/getting-started-kubernetes.rst

```diff
@@ -1963,7 +1978,6 @@ Prerequisites for Worker Nodes
 Supported operating systems:
 
 * Ubuntu 22.04
-* RHEL 8.8
```

Comment:
I don't know why it was added here. AFAIK this feature was never tested on RHEL 8.8